test/utils.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * utils.js
+ *
+ * Copyright © 2023 Box, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Remove a common indent from each line of a multiline string. Exact indent
+ * sequence is based on the indent of first non-empty line. Lines which don't
+ * begin with the detected indent are not modified.
+ */
```

Comment:
Add in jsdocs for params and return value

Reply:
Fixed